### DISCUSSION TITLE
Lammps build fails with MPI link problem

### DISCUSSION THREAD

**Discussion by Mskater** [deepmodeling/deepmd-kit#293] [Q&A]
I did install openmpi and when trying to build Lammps with make mpi -j4 got the following error:

mpicxx: error while loading shared libraries: libopen-pal.so.40: cannot open shared object file: No such file or directory
Makefile:89: recipe for target '../lmp_mpi' failed

I'm on Ubuntu 18.04

I'm sure I just don't know enough of these simple things, I appreciate all the help!

**Comment by njzjz** [comment]
1. check if `libopen-pal.so.40` in your openmpi directory e.g. `openmpi/lib`?
2. check if you append `openmpi/lib` to `LD_LIBRARY_PATH`?

**Comment by Mskater** [comment]
Thanks, now I got over the library link part.
